src/components/find-on-shelf/FindOnShelfManifestationList.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { FC } from "react";
+import { totalAvailableMaterialsInBranch } from "../../apps/material/helper";
+import { ManifestationsSimpleFieldsFragment } from "../../core/dbc-gateway/generated/graphql";
+import { HoldingsV3 } from "../../core/fbs/model";
+import { useText } from "../../core/utils/text";
+import FindOnShelfManifestationListItem from "./FindOnShelfManifestationListItem";
+
+export interface FindOnShelfManifestationListProps {
+  holding: HoldingsV3;
+  title: string;
+  manifestations: ManifestationsSimpleFieldsFragment[];
+}
+
+const FindOnShelfManifestationList: FC<FindOnShelfManifestationListProps> = ({
+  holding,
+  title,
+  manifestations
+}) => {
+  const t = useText();
+  const { materials, department, location, sublocation } = holding;
+  const numberAvailable = totalAvailableMaterialsInBranch(materials);
+
+  return (
+    <ul className="find-on-shelf">
+      <li className="find-on-shelf__header-row text-small-caption">
+        <span className="find-on-shelf__material-header">
+          {t("findOnShelfModalListMaterialText")}
+        </span>
+        <span>{t("findOnShelfModalListFindOnShelfText")}</span>
+        <span className="find-on-shelf__item-count-header">
+          {t("findOnShelfModalListItemCountText")}
+        </span>
+      </li>
+      {manifestations.map((manifestation) => {
+        return (
+          <FindOnShelfManifestationListItem
+            department={department?.title}
+            location={location?.title}
+            sublocation={sublocation?.title}
+            title={title}
+            material={materials[0]}
```

Comment:
Please look into whether you are in fact handling the relationship between manifestations and holdings correctly. Always matcing the same holdings to each manifestation does not seem right.

Reply:
Correct-o! It was incorrect. 
This was one hell of a task, especially because we cannot yet retrieve the correct data to show. Or rather to transform and then show it. But thanks for all the help and suggestions IRL. ❤️ 
I believe we now show all this correctly, but you're welcome to correct me if I oversaw anything.